packages/run-protocol/src/runStake/runStakeManager.js

```diff
@@ -0,0 +1,168 @@
+// @ts-check
+import { AmountMath } from '@agoric/ertp';
+import { floorMultiplyBy } from '@agoric/zoe/src/contractSupport/index.js';
+import { makeRatio } from '@agoric/zoe/src/contractSupport/ratio.js';
+import { fit, getCopyBagEntries, M } from '@agoric/store';
+import { makeNotifierKit, observeNotifier } from '@agoric/notifier';
+import { E } from '@endo/far';
+import { makeTracer } from '../makeTracer.js';
+import { chargeInterest } from '../interest.js';
+import { KW } from './runStakeKit.js';
+
+const { details: X } = assert;
+
+const trace = makeTracer('RM'); // TODO: how to turn this off?
+
+/**
+ * @param {ZCF} zcf
+ * @param {ZCFMint<'nat'>} debtMint
+ * @param {{ debt: Brand<'nat'>, Attestation: Brand<'copyBag'>, Stake: Brand<'nat'> }} brands
+ * @param {*} paramManager
```

Comment:
haven't I heard that declaring it `unknown` would let the types flow through?

Reply:
no, not correctly, anyway. To make types flow through, I think type parameters are necessary.